ElectronicObserver/Window/Tools/EquipmentUpgradePlanner/EquipmentUpgradePlanManager.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using ElectronicObserver.Data;
+using ElectronicObserver.Database;
+using ElectronicObserver.Observer;
+using ElectronicObserverTypes;
+using ElectronicObserverTypes.Mocks;
+
+namespace ElectronicObserver.Window.Tools.EquipmentUpgradePlanner;
+public class EquipmentUpgradePlanManager
+{
+	public bool IsInitialized { get; private set; } = false;
+
+	public ObservableCollection<EquipmentUpgradePlanItemViewModel> PlannedUpgrades { get; private set; } = new();
+
+	public EquipmentUpgradePlanManager()
+	{
+		SubscribeToApi();
```

Comment:
I'd just remove `SubscribeToApi` and call `Load` here.

Reply:
If i call load on construct the KCDatabase.Instance.MasterEquipments is not populated yet, thats why i subscribed to port API (and will subscribe to other API in the future)

Reply:
Yeah, I was thinking this would only get used as a tool, which can't get opened before the equipment gets loaded anyway. But since it's also meant to be a view, this code makes sense.